ChangeLog.md

```diff
@@ -4619,6 +4619,7 @@ Supported -EnableNoPublicIP when creating a Databricks workspace
     - 'Add-AzServiceFabricManagedNodeTypeVMSecret'
     - 'Remove-AzServiceFabricManagedNodeTypeVMExtension'
     - 'Restart-AzServiceFabricManagedNodeTyp'
+    - 'Add-AzServiceFabricManagedNetworkSecurityRule'
```

Comment:
these are the release notes from 5.7.0 - March 2021 - should remove this edit

Reply:
believe this will get populated from the change logs of each cmdlet lib

Reply:
You should not change this file. Please add changelog to the changelog.md under you module folder.

Reply:
@mwesigwaguma  remove this change please